contracts/src/v0.8/dev/KeeperRegistryDev.sol

```diff
@@ -256,6 +256,25 @@ contract KeeperRegistryDev is
     _fallback();
   }
 
+  /**
+   * @notice proposes the safe transfer of an upkeep's admin role to another address
+   * @param id the upkeep id to transfer admin
+   * @param proposed address to nominate for the new upkeep admin
+   */
+  function transferUpkeepAdmin(uint256 id, address proposed) external {
```

Comment:
can you add these functions to interface?

Reply:
Done